openwisp_controller/connection/connectors/snmp.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from jsonschema import validate
+
+logger = logging.getLogger(__name__)
+
+
+class Snmp(object):
+    schema = {
+        '$schema': 'http://json-schema.org/draft-04/schema#',
+        'type': 'object',
+        'title': 'Credentials type',
+        'oneOf': [
```

Comment:
doesn't seem to me we need to use `oneOf` here do we? Also the title above I'd remove it.